_data/simple-icons.json

```
@@ -8279,6 +8279,11 @@
             "source": "https://brand.opera.com/1472-2/opera-logos/",
             "guidelines": "https://brand.opera.com"
         },
+        {
+            "title": "Opera GX",
+            "hex": "EE2950",
+            "source": "https://www.opera.com/gx"
```

```suggestion
            "source": "https://brand.opera.com/1472-2/opera-logos/"
```

The monochrome logo for Opera GX is specifically available at the logos page you'd previously used. Can you please leave that as the source.